doc/source/vertical_ref.md

```diff
@@ -12,15 +12,64 @@ kernelspec:
 ---
 (vertical-ref)=
 
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+# To get a good resolution for displayed figures
+from matplotlib import pyplot
+pyplot.rcParams['figure.dpi'] = 600
+pyplot.rcParams['savefig.dpi'] = 600
+```
+
 # Vertical referencing
 
-xDEM supports the use of **vertical coordinate reference systems (vertical CRSs)** and vertical transformations for DEMs
+xDEM supports the use of **vertical coordinate reference systems (vertical CRSs) and vertical transformations for elevation data**
 by conveniently wrapping PROJ pipelines through [Pyproj](https://pyproj4.github.io/pyproj/stable/) in the {class}`~xdem.DEM` class.
 
-```{important}
+```{note}
 **A {class}`~xdem.DEM` already possesses a {class}`~xdem.DEM.crs` attribute that defines its 2- or 3D CRS**, inherited from
 {class}`~geoutils.Raster`. Unfortunately, most DEM products do not yet come with a 3D CRS in their raster metadata, and
```

Comment:
Maybe it can be good also to explain that the 2D CRS may contain a term "datum" that is misleading. Even for 2D projections, one needs a datum, which is very often WGS84, for projecting from degrees to meters, but this is not the same as the vertical datum. There was some confusion about it at the winter school.

Reply:
For example, projection UTM zone 44N (EPSG:32644), if printed as WKT string, starts with:
PROJCS["WGS 84 / UTM zone 44N",
    GEOGCS["WGS 84",
        **DATUM["WGS_1984",**
            SPHEROID["WGS 84",6378137,298.257223563,
                AUTHORITY["EPSG","7030"]],
            AUTHORITY["EPSG","6326"]],

Reply:
A future improvement that would fit better on the guide pages, the feature pages should not get too much into details.

Reply:
:wink: